kalite/inline/api_views.py

```diff
@@ -0,0 +1,18 @@
+"""
+Views accessible as an API endpoint for inline.  All should return JsonResponses.
+"""
+import yaml
+import json
+
+from fle_utils.internet.classes import JsonResponse
+
+
+def narrative_view(request, narrative_id):
+    f = open("kalite/inline/narratives.yaml", 'r')
+    narrative_json = yaml.safe_load(f)
+    f.close()
+
+
+    print "!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!API_VIEWS!!!!!"
+    print narrative_json
+    return JsonResponse(narrative_json)
```

Comment:
At any rate, if you choose to implement a different scheme this view will need to pick the appropriate narrative using the `narrative_id` parameter.